README.md

```diff
@@ -17,13 +17,13 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
-[] Undefined
+[X] Strings
+[X] Booleans
+[X] Undefined
 [] NaN
-[] Integers
+[X] Integers
```

Comment:
JavaScript only has `Number`.  It does not define different types of numbers, like integers, short, long, floating-point etc.